spec/api_spec.rb

```diff
@@ -31,6 +71,16 @@
       })
     end
 
+    it 'builds user_attributes_hash with device' do
+      user = users.first.merge(devices: devices)
+      user_attributes_hash = api.send(:build_user_attributes_hash, user)
+      expect(user_attributes_hash[:devices].first).to eq({ device_id: 'fu123',
```

Comment:
sorry there's no hound on this repo, but this formatting is not ok.
1. shouldn't this just be `devices.first.with_indifferent_access`?
2. if not, format like this
```
expect(user_attributes_hash[:devices].first).to eq({
  device_id: 'fu123',
  appVersion: 'x42x',
  deviceModel: 'p0d',
  create_date: '2018-01-01'.to_date
}.with_indifferent_access)
```